docs/content/Schema/Fundamentals/Concepts.md

```diff
@@ -0,0 +1,323 @@
+
+**`line_items`**
+
+| **id** | **product_id** | **order_id** | **quantity** | **price** | **created_at**             |
+| ------ | -------------- | ------------ | ------------ | --------- | -------------------------- |
+| 1      | 31             | 1            | 1            | 275       | 2019-01-31 00:00:00.000000 |
+| 2      | 49             | 2            | 6            | 248       | 2021-01-20 00:00:00.000000 |
+| 3      | 89             | 3            | 6            | 197       | 2021-11-25 00:00:00.000000 |
+| 4      | 71             | 4            | 8            | 223       | 2019-12-23 00:00:00.000000 |
+| 5      | 64             | 5            | 5            | 75        | 2019-04-20 00:00:00.000000 |
+
+## Cubes
+
+A cube represents a dataset in Cube.js, and is conceptually similar to a [view
+in SQL][wiki-view-sql]. Cubes are typically declared in separate files with one
+cube per file. Within each cube are definitions of [dimensions][self-dimensions]
+and [measures][self-measures]. Typically, a cube points to a single table in
+your database using the [`sql` property][ref-schema-ref-sql]:
+
+```javascript
+cube('Orders', {
+  sql: `SELECT * FROM orders`,
+});
+```
+
+The `sql` property of a cube is flexible enough to accommodate more complex SQL
+queries too:
+
+```javascript
+cube('Orders', {
+  sql: `
+SELECT
+  *
+FROM
+  orders,
+  line_items
+WHERE
+  orders.id = line_items.order_id
+  `,
+});
+```
+
+## Dimensions
+
+Dimensions are [non-additive][ref-caching-nonadditive], or
+[qualitative][wiki-qualitative] properties of the cube; they are often
+non-numeric, and mathematical operations such as `SUM()` or `MIN()` **cannot**
+be performed on them. In practice, dimensions often represent information such
+as colors, URLs or SKUs.
+
+[The `orders` table](#top) contains only dimensions, so representing them in the
+`Orders` cube is straightforward:
+
+```javascript
+cube('Orders', {
+
+  ...,
+
+  dimensions: {
+    id: {
+      sql: `id`,
+      type: `number`,
+      // Here we explicitly let Cube.js know this field is the primary key
+      // This is required for de-duplicating results when using joins
+      primaryKey: true
+    },
+    status: {
+      sql: `status`,
+      type: `string`
+    },
+  },
+});
+```
+
+[The `line_items` table](#top) also has a couple of dimensions which can be
+represented as follows:
+
+```javascript
+cube('LineItems', {
+
+  ...,
+
+  dimensions: {
+    id: {
+      sql: `id`,
+      type: `number`,
+      // Again, we explicitly let Cube.js know this field is the primary key
+      // This is required for de-duplicating results when using joins
+      primaryKey: true
+    },
+  },
+});
+```
+
+Dimensions can be of different types, and you can find them all
+[here][ref-schema-dimension-types].
+
+### Time Dimensions
+
+<!-- prettier-ignore-start -->
+[[info | ]]
+| Time-based properties are generally used as dimensions in Cube.js. // say why
```

Comment:
Is there a specific reason temporal attributes in a dataset are dimensions? I would imagine its because timestamps aren't quite measures (although that's not _completely_ true either). Would <3 some clarification

/cc @keydunov @paveltiunov